plugins/sudo.js

```
@@ -1,48 +1,87 @@
+  {
+    pattern: 'setsudo',
+    public: false,
+    desc: 'SetSudo A Specific User',
+    type: 'sudo',
+  },
+  async (message, match) => {
+    const jid = await message.getUserJid(match);
+    if (!jid) return;
+    if (isSudo(jid)) return message.send('_Already A Sudo User_');
+    addSudo(jid);
+    return message.send('```@' + jid + ' is now a Sudo```', { mentions: [jid] });
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)